### ISSUE TITLE
Selecting Raptor Repeatedly

### ISSUE THREAD

**Issue by evilandrex** [Friendly0Fire/GW2Addons#34]
When I select the raptor (the top most position in the menu), it repeatedly selects it until I move my mouse into another mount's position.
https://www.youtube.com/watch?v=Wfcu_FxdBSQ

**Comment by Friendly0Fire** [comment]
Your overlay key is Z and your Raptor key is Alt+Z. Try to set it to anything but Z and see if that works.

**Comment by evilandrex** [comment]
Yes, that was indeed the issue. This was not the case on the previous version but the solution is usable.

**Comment by Friendly0Fire** [comment]
Alright, good. That means I can look into what's causing it now that we've narrowed it down.
